### PR TITLE
8.7 Release Notes

### LINKED ISSUE #2884: 8.7.0 Release notes
### Description
Generate the 8.6.0 release notes and publish them before or on the release day, which is **February 7, 2023**.

### Milestone dates
- **FF:** February 7, 2023
- **BC1:** February 8, 2023
- **Release:** March 28, 2023

#### Prep/admin
- [x] Open a doc issue for Endpoint release notes (can model the issue after https://github.com/elastic/security-docs/issues/2685)- https://github.com/elastic/security-docs/issues/2983
- [x] Add the release notes PR to the docs release issue - https://github.com/elastic/dev/issues/2214
- [x] Set up the breaking changes section within the 8.7 release notes 
- [x] Create a doc issue (optional) and PR for the breaking changes section in the Stack docs: https://github.com/elastic/stack-docs/issues/2351

#### Reviews
- [ ] Verify all Endpoint PRs are release-noted (check https://github.com/elastic/security-docs/issues/2983) 
- [ ] Complete the technical review 
   - [ ] Ensure all breaking changes are doc'd
   - [ ] Ensure all known issues and workarounds are doc'd 
- [ ] Complete the editorial review 

#### Publishing
- [ ] Merge the Security release notes PR 
- [ ] Merge the Stack docs PR 

### Notes
- https://github.com/elastic/kibana/pull/146373 - Only needs release notes
- https://github.com/elastic/kibana/pull/147628 - Possible summary: Removes fields that are not ECS-compliant before creating alerts. 
- https://github.com/elastic/kibana/pull/150181 - Don't need to release-note. This is flagged to be skipped, is part of a feature that hasn't been released yet, and doesn't present any notable user-facing changes.
- [x] @dhurley14 to create a doc issue that describes the following known issue: https://github.com/elastic/kibana/issues/150818 
   - [x] Document https://github.com/elastic/security-docs/issues/3046 as a known issue for 8.7. 
- [x] As per @MikePaquette's comment [here](https://github.com/elastic/security-team/issues/5766#issuecomment-1460989588), will need to mention CCS support for indicator match rules.
- [x] Will need to release-note https://github.com/elastic/kibana/pull/151952 as an enhancement.

### LINKED ISSUE #2983: 8.7 Endpoint release notes
@roxana-gheorghe @magermark @jtnk @nfritts @softengchick 

Filing this doc issue to track Endpoint features that will need to be documented in the 8.7 Release Notes.

### LINKED ISSUE #3046: [Release Notes][BUG] [8.7] Sourcerer will display the `.alerts` index in the matched index patterns section after refreshing page
**Description**:

Sourcerer will only query index patterns where indices that match the given index pattern are found. The `.alerts-security.alerts-{spaceId}` index, which is part of the security solution default data view for sourcerer, is not created until an alert is written (which happens when a user has a rule run to fulfillment.) Because the `.alerts` index doesn't exist until after an alert is written, sourcerer is not aware that there now exists an index that matches the `.alerts` portion of the data view. That means that the `.alerts` index is not included as a data source when navigating between pages. This is only an issue for customers that have never had an alert written by one of their rules, and navigates to something like timeline to query for their alerts. The solution is to refresh the web browser. Sourcerer will refetch the matched indices and present the `.alerts` index as a pattern that can be queried. This is a very rare circumstance for our customers and the solution is very simple so hopefully not a big disruption to customer workflows.

<details><summary>What to look for</summary>

Rule runs, navigate to timeline, click on data view dropdown (sourcerer) no .alerts index

before_screenshot

refresh the browser and the `.alerts` index is now available for query:

after_screenshot

</details>

cc: @nastasha-solomon

### PR DESCRIPTION
**Pull request by benironside**
Fixes #2884, fixes #3046, and fixes #2983 by creating release notes for version 8.7

Something to think about: traditionally we've had a section called "bug fixes and enhancements". What do you think of separating enhancements from bug fixes?  

Preview: [8.7 release notes ](https://security-docs_3049.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.7.0.html)

### CODE REVIEW

File: docs/release-notes/8.7.asciidoc [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+[[deprecations-8.7.0]]
+==== Deprecations
+There are no deprecations in 8.7.0.
+
+
+[discrete]
+[[features-8.7.0]]
+==== Features
+* Creates a new dashboard, Data Quality, which highlights any issues in your ECS field mappings ({pull}150063[#150063]).
+* Creates a Torq connector that can trigger Torq workflows ({pull}149405[#149405]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.7.asciidoc:34]: This needs to be added to our docs (I'll file a doc issue for it) and I believe it needs to be added to the kibana release notes. I'll check with Kaarina and Lisa before I add it.

Comment (nastasha-solomon) [reply docs/release-notes/8.7.asciidoc:34]: Will keep in RN but won't mention in release highlights.

**Review by kqualters-elastic — APPROVED** [review]

File: docs/release-notes/8.7.asciidoc [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+* Improves data fetch performance throughout Elastic Security, especially for deployments with large indices and many integrations ({pull}142904[#142904]).
+
+* Fixes a bug that could cause your cursor to jump to the end of the text field when editing a rule action message ({pull}150823[#150823]).
+* Fixes a bug that could result in incorrect links to machine learning jobs from search results ({pull}150881[#150881]).
+* Fixes a bug that caused a fade in / fade out effect on alert descriptions ({pull}150998[#150998]).
+* Fixes a bug that caused the Alerts page to default to the wrong chart type ({pull}151073[#151073]).
+* Fixes a bug that could hide some shared exception lists when you changed the number of rows in the exceptions lists table ({pull}151073[#151073]).
+* Removes a blank option from the Add rule exception page's Field browser ({pull}151398[#151398]).
+* Fixes a bug with the empty state that appears when your exception lists search yields no results ({pull}151530[#151530]).
+* Fixes a bug where UI text conflated endpoint exceptions with rule exceptions on the exception and rule management pages ({pull}151532[#151532]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.7.asciidoc:72]: The second part of this sentence is a bit wordy. Would the following work?
 
```suggestion
* Fixes a UI text bug that conflated Endpoint exceptions with regular rule exceptions ({pull}151532[#151532]).
```

File: docs/release-notes/8.7.asciidoc [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+* Fixes a bug that could cause an unnecessary warning to display on the Add rule exception menu ({pull}151570[#151570]).
+* Fixes a bug that sometimes prevented a timeline from saving when it was created using the "Investigate in timeline" action on an alert ({pull}151616[#151616]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.7.asciidoc:74]:
```suggestion
* Fixes a bug that sometimes prevented a Timeline from saving when it was created using the *Investigate in timeline* action on an alert ({pull}151616[#151616]).
```

File: docs/release-notes/8.7.asciidoc [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+* Fixes a bug that could result in unnecessary validation errors in text entry fields on the Add rule exception menu ({pull}151654[#151654]).
+* Fixes a bug that caused some module names to be partially hidden on the Overview dashboard ({pull}151843[#151843]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.7.asciidoc:76]:
```suggestion
* Fixes a bug that caused some module names to be partially hidden in the Overview dashboard ({pull}151843[#151843]).
```

File: docs/release-notes/8.7.asciidoc [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+* Fixes a visual bug that affected empty Rule previews ({pull}151869[#151869]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.7.asciidoc:77]:
```suggestion
* Fixes a visual bug that affected empty rule previews ({pull}151869[#151869]).
```

File: docs/release-notes/8.7.asciidoc [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+* Fixes a bug that could cause alert details to incorrectly show installed integrations as not installed ({pull}152055[#152055]).
+* Changed the Import value lists page's "Import list" button to say "Import value list" ({pull}152281[#152281]).
+* Improves the toast message that appears when you export an exception list ({pull}152301[#152301]).
+* Fixes a bug that broke Analyzer for Sysmon data ingested via {agent} ({pull}152418[#152418]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.7.asciidoc:81]:
```suggestion
* Fixes a bug that broke the visual analyzer for sysmon data ingested via {agent} ({pull}152418[#152418]).
```

File: docs/release-notes/8.7.asciidoc [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+* Fixes a bug that incorrectly allowed you to use custom fields in the Add Endpoint Exception menu ({pull}152619[#152619]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.7.asciidoc:82]:
```suggestion
* Fixes a bug that incorrectly allowed you to use custom fields in the Add Endpoint Exception flyout ({pull}152619[#152619]).
```

**Review by nastasha-solomon — COMMENTED** [review]
Thanks for putting all of this together, @benironside! Another huge effort! I suggested a few revisions, but overall everything looks great. Thanks!

**Review by nastasha-solomon — COMMENTED** [review]
Thanks for putting all of this together, @benironside! Another huge effort! I suggested a few revisions, but overall everything looks great. Thanks!

File: docs/release-notes/8.7.asciidoc [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+== 8.7
+
+[discrete]
+[[release-notes-8.7.0]]
+=== 8.7.0
+
+[discrete]
+[[known-issue-8.7.0]]
+==== Known issues
+* After your instance of {elastic-sec} generates alerts for the first time, you may have to refresh your browser before your alert data will appear (navigating between pages will not suffice) (https://github.com/elastic/security-docs/issues/3046[#3046]).
```
[the remark this answers is not in the document]
Comment (dhurley14) [reply docs/release-notes/8.7.asciidoc:11]: your suggestions look perfect thank you!

**Review by dhurley14 — APPROVED** [review]

File: docs/release-notes/8.7.asciidoc [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+[discrete]
+[[features-8.7.0]]
+==== Features
+* Creates a new dashboard, Data Quality, which highlights any issues in your ECS field mappings ({pull}150063[#150063]).
+* Introduces a new event type, `Credential access` to represent credential dumping attempts on Windows, such as by tools like Mimikatz or fgdump.
+* Creates a Torq connector that can trigger Torq workflows ({pull}149405[#149405]).
+* Adds more key performance indicator charts to the Alerts page ({pull}150242[#150242], {pull}149173[#149173], and {pull}146938[#146938]).
+* Adds filters to the Alerts page ({pull}146989[#146989]).
+* Enables you to set expiration dates for rule exceptions, and to choose whether to include expired exceptions when you export exception lists ({pull}145180[#145180]).
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/release-notes/8.7.asciidoc:38]:
```suggestion
* Enables you to set expiration dates for rule exceptions and choose whether to include expired exceptions when you export shared exception lists ({pull}145180[#145180]).
```

File: docs/release-notes/8.7.asciidoc [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+* Fixes a bug that caused a fade in and out effect on alert descriptions ({pull}150998[#150998]).
+* Fixes a bug that caused the Alerts page to default to the wrong chart type ({pull}151073[#151073]).
+* Fixes a bug that could hide some shared exception lists when you changed the number of rows in the exceptions lists view ({pull}151393[#151393]).
+* Removes a blank option from the *Field* browser in the Add rule exception flyout ({pull}151398[#151398]).
+* Fixes a bug with the empty state that appears when your exception lists search yields no results ({pull}151530[#151530]).
+* Fixes a UI text bug that conflated Endpoint exceptions with regular rule exceptions ({pull}151532[#151532]).
+* Fixes a bug that could cause an unnecessary warning to display in the Add rule exception flyout ({pull}151570[#151570]).
+* Fixes a bug that sometimes prevented a Timeline from saving when it was created using the *Investigate in timeline* action on an alert ({pull}151616[#151616]).
+* Fixes a bug that could result in unnecessary validation errors in text entry fields in the Add rule exception flyout ({pull}151654[#151654]).
```
Comment (jmikell821) [inline docs/release-notes/8.7.asciidoc:74]:
```suggestion
* Fixes a bug that could cause unnecessary validation errors in text entry fields in the Add rule exception flyout ({pull}151654[#151654]).
```

File: docs/release-notes/8.7.asciidoc [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+* Allows placeholder fields in Osquery queries ({pull}146598[#146598]).
+* Allows the Rules table state to persist even after you refresh or navigate to another page ({pull}145111[#145111]).
+* Improves data fetch performance throughout {elastic-sec}, especially for deployments with large indices and many integrations ({pull}142904[#142904]).
+* Fixes a bug that could cause your cursor to jump to the end of the text field when editing a rule action message ({pull}150823[#150823]).
+* Fixes a bug that could result in incorrect links to machine learning jobs from search results ({pull}150881[#150881]).
+* Fixes a bug that caused a fade in and out effect on alert descriptions ({pull}150998[#150998]).
+* Fixes a bug that caused the Alerts page to default to the wrong chart type ({pull}151073[#151073]).
+* Fixes a bug that could hide some shared exception lists when you changed the number of rows in the exceptions lists view ({pull}151393[#151393]).
+* Removes a blank option from the *Field* browser in the Add rule exception flyout ({pull}151398[#151398]).
```
Comment (jmikell821) [inline docs/release-notes/8.7.asciidoc:69]: Can we put all the "Add rule exception flyout" bugs together one after the other? (Lines 69, 72, and 74).

**Review by jmikell821 — COMMENTED** [review]
Thanks @benironside and @nastasha-solomon for pulling these together! Just a few comments. Would like to review once more with fresh eyes before we merge. Thanks!

File: docs/release-notes/8.7.asciidoc [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+
+
+[discrete]
+[[features-8.7.0]]
+==== Features
+* Creates a new dashboard, Data Quality, which highlights any issues in your ECS field mappings ({pull}150063[#150063]).
+* Introduces a new event type (`Credential access`) to represent credential dumping attempts on Windows by tools like Mimikatz or fgdump.
+* Creates a Torq connector that can trigger Torq workflows ({pull}149405[#149405]).
+* Adds more key performance indicator charts to the Alerts page ({pull}150242[#150242], {pull}149173[#149173], and {pull}146938[#146938]).
+* Adds filters and new key performance indicator charts to the Alerts page ({pull}146989[#146989], {pull}146938[#146938]).
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/release-notes/8.7.asciidoc:37]: Yep, it is. I'll revert my change and remove 146938 from line 37.

```suggestion
* Adds filters to the Alerts page ({pull}146989[#146989], {pull}146938[#146938]).
```

**Comment by nastasha-solomon** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/release-notes/8.7.asciidoc [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+
+
+[discrete]
+[[features-8.7.0]]
+==== Features
+* Creates a new dashboard, Data Quality, which highlights any issues in your ECS field mappings ({pull}150063[#150063]).
+* Introduces a new event type (`Credential access`) to represent credential dumping attempts on Windows by tools like Mimikatz or fgdump.
+* Creates a Torq connector that can trigger Torq workflows ({pull}149405[#149405]).
+* Adds more key performance indicator charts to the Alerts page ({pull}150242[#150242], {pull}149173[#149173], and {pull}146938[#146938]).
+* Adds filters to the Alerts page ({pull}146989[#146989]).
```
Comment (joepeeples) [inline docs/release-notes/8.7.asciidoc:37]: The Alerts page filters are behind a feature flag for 8.7, so the feature has essentially been pushed back to 8.8.

File: docs/release-notes/8.7.asciidoc [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+[[bug-fixes-8.7.0]]
+==== Bug fixes and enhancements
+* Various bug fixes and UX enhancements for the Alerts page ({pull}152402[#152402]).
+* Improves the formatting and readability of machine learning job names ({pull}148974[#148974], {pull}148780[#148780]).
+* Introduces cross-cluster search support for Indicator Match rules by improving rule performance ({pull}149113[#149113]).
+* Warns you about type conflicts and unmapped indices when you're creating rule exceptions ({pull}149149[#149149]).
+* Improves sorting of the Rules table: allows you to sort it by any column, removes the *Advanced sorting* toggle, and removes the `Version` column ({pull}149840[#149840]).
+* Adds a *Clear table filters* button to the Rules page so you can clear all filters in one click ({pull}150059[#150059]).
+* Adds buttons to the Rules page that allow you to view only enabled or disabled rules ({pull}150153[#150153]).
+* The *Related alerts by process ancestry* section of the alert details flyout is now GA ({pull}152011[#152011]).
```
Comment (joepeeples) [inline docs/release-notes/8.7.asciidoc:55]:
```suggestion
* The *Related alerts by process ancestry* section of the alert details flyout is now generally available (GA) ({pull}152011[#152011]).
```

**Review by joepeeples — COMMENTED** [review]
Awesome, massive work! Just two small comments

File: docs/release-notes/8.7.asciidoc [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+* Adds the option to suppress custom query rule alerts during a specific time window. Duplicate alerts within that time window will be grouped ({pull}148868[#148868]).
```
Comment (peluja1012) [inline docs/release-notes/8.7.asciidoc:56]: Hi @benironside, could we please move this one to the `Features` section?

File: docs/release-notes/8.7.asciidoc [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+* Removes a blank option from the *Field* browser in the Add rule exception flyout ({pull}151398[#151398]).
+* Fixes a UI text bug that conflated Endpoint exceptions with regular rule exceptions ({pull}151532[#151532]).
+* Fixes a bug that could cause an unnecessary warning to display in the Add rule exception flyout ({pull}151570[#151570]).
+* Fixes a bug with the empty state that appears when your exception lists search yields no results ({pull}151530[#151530]).
+* Fixes a bug that sometimes prevented a Timeline from saving when it was created using the *Investigate in timeline* action on an alert ({pull}151616[#151616]).
+* Fixes a bug that could cause unnecessary validation errors in text entry fields in the Add rule exception flyout ({pull}151654[#151654]).
+* Fixes a bug that caused some module names to be partially hidden on the Overview dashboard ({pull}151843[#151843]).
+* Fixes a visual bug that affected empty rule previews ({pull}151869[#151869]).
+* Fixes a bug that could cause alert details to incorrectly show installed integrations as not installed ({pull}152055[#152055]).
```
Comment (banderror) [inline docs/release-notes/8.7.asciidoc:77]: I think by now we should be using the term `rule` instead of `alert` everywhere, at least in the app we renamed it a long time ago. Do we still call detection rules "alerts" in the docs?

Another thing is that the related integrations UI is displayed not on the Rule Details page, but also in the Rules table in each row. So I'd reformulate this a bit:

`Fixes a bug that could cause rule's related integrations that are installed to be incorrectly shown as not installed in the Rules table and on the rule details page`

We can also add https://github.com/elastic/kibana/pull/149646 to the same bullet point -- it addresses the same bug.

File: docs/release-notes/8.7.asciidoc [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+* Simplifies the interface for navigating from a rule's details page back to the Rules page ({pull}147357[#147357]).
+* Allows you to resize the Rule preview panel ({pull}147351[#147351]).
+* Improves the Bulk Edit API by adding a `skipped` property to rules that weren't updated, and updates the toast message to show which were skipped ({pull}147345[#147345]).
+* Allows placeholder fields in Osquery queries ({pull}146598[#146598]).
+* Allows the Rules table state to persist even after you refresh or navigate to another page ({pull}145111[#145111]).
+* Improves data fetch performance throughout {elastic-sec}, especially for deployments with large indices and many integrations ({pull}142904[#142904]).
+* Fixes a bug that could cause your cursor to jump to the end of the text field when editing a rule action message ({pull}150823[#150823]).
+* Fixes a bug that could result in incorrect links to machine learning jobs from search results ({pull}150881[#150881]).
+* Fixes a bug that caused a fade in and out effect on alert descriptions ({pull}150998[#150998]).
```
Comment (banderror) [inline docs/release-notes/8.7.asciidoc:66]: Same comment here -- probably should be called "rule description".

**Review by banderror — COMMENTED** [review]
Awesome, thanks @benironside! Just a few comments, including a general one below.

This PR is not mentioned in the release notes: https://github.com/elastic/kibana/pull/147806 - do we have it in some 8.6 notes or decided to skip it?

File: docs/release-notes/8.7.asciidoc (hunk 16 above)
Comment (benironside) [reply docs/release-notes/8.7.asciidoc:77]: Thanks for the review Georgii! We definitely don't use alerts to refer to detection rules, so I think you're right about this being incorrect. Thank you for the excellent edit.

File: docs/release-notes/8.7.asciidoc [hunk 18]
```diff
@@ -0,0 +1,84 @@
...
+[[deprecations-8.7.0]]
+==== Deprecations
+There are no deprecations in 8.7.0.
+
+
+[discrete]
+[[features-8.7.0]]
+==== Features
+* Creates a new dashboard, Data Quality, which highlights any issues in your ECS field mappings ({pull}150063[#150063]).
+* Introduces a new event type (`Credential access`) to represent credential dumping attempts on Windows by tools like Mimikatz or fgdump.
```
Comment (jmikell821) [inline docs/release-notes/8.7.asciidoc:34]:
```suggestion
* Introduces a new event type (`Credential access`) to represent credential dumping attempts on Windows using tools like Mimikatz or fgdump.
```

File: docs/release-notes/8.7.asciidoc [hunk 19]
```diff
@@ -0,0 +1,84 @@
...
+* The *Related alerts by process ancestry* section of the alert details flyout is now generally available (GA) ({pull}152011[#152011]).
+* Adds the option to suppress custom query rule alerts during a specific time window. Duplicate alerts within that time window will be grouped ({pull}148868[#148868]).
+* Reduces alert creation errors by stopping the detection engine from writing non-ECS-compliant fields to alerts from source events ({pull}147628[#147628]).
+* Simplifies the interface for navigating from a rule's details page back to the Rules page ({pull}147357[#147357]).
+* Allows you to resize the Rule preview panel ({pull}147351[#147351]).
+* Improves the Bulk Edit API by adding a `skipped` property to rules that weren't updated, and updates the toast message to show which were skipped ({pull}147345[#147345]).
+* Allows placeholder fields in Osquery queries ({pull}146598[#146598]).
+* Allows the Rules table state to persist even after you refresh or navigate to another page ({pull}145111[#145111]).
+* Improves data fetch performance throughout {elastic-sec}, especially for deployments with large indices and many integrations ({pull}142904[#142904]).
```
Comment (jmikell821) [inline docs/release-notes/8.7.asciidoc:63]: Nit - but "several" sounds a bit better here, IMO. 
```suggestion
* Improves data fetch performance throughout {elastic-sec}, especially for deployments with large indices and several integrations ({pull}142904[#142904]).
```

File: docs/release-notes/8.7.asciidoc [hunk 20]
```diff
@@ -0,0 +1,84 @@
...
+* Fixes a bug that caused some module names to be partially hidden on the Overview dashboard ({pull}151843[#151843]).
+* Fixes a visual bug that affected empty rule previews ({pull}151869[#151869]).
+* Fixes a bug that could cause a rule's related integrations to incorrectly appear as not installed on the Rules table and the rule details page ({pull}152055[#152055], {pull}149646[#149646]).
+* Changes the *Import list* button name to *Import value list* ({pull}152281[#152281]).
+* Improves the toast message that appears when you export an exception list ({pull}152301[#152301]).
+* Fixes a bug that broke the visual analyzer for sysmon data ingested via {agent} ({pull}152418[#152418]).
+* Fixes a bug that incorrectly allowed you to use custom fields in the Add Endpoint Exception flyout ({pull}152619[#152619]).
+* Fixes a bug where the two breadcrumbs on shared exception lists pages did not use the same text ({pull}152629[#152629]).
+* Fixes an issue in the Update exception item API that incorrectly merged existing objects with updated objects ({pull}151952[#151952]).
+* Fixes a bug affecting the rule status refresh loading indicator ({pull}147806[#147806]).
```
Comment (jmikell821) [inline docs/release-notes/8.7.asciidoc:84]:
```suggestion
* Fixes a bug that affected the rule status refresh loading indicator ({pull}147806[#147806]).
```

**Review by jmikell821 — COMMENTED** [review]
Few small nits. Thanks for pulling these together!

**Review by banderror — APPROVED** [review]
Thank you!

**Review by michaelolo24 — APPROVED** [review]
Looks great on the investigations changes!
